game/functions/functions_server.hpp

```diff
@@ -355,4 +356,25 @@ class vgm_s
         class skills_handle_skillRespecRequest {};
         class skills_handle_skillsDataRequest {};
     };
+
+    class virtual_squads
+    {
+        VGM_SERVER_PATH(\systems\virtual_squads\server);
+
+        class virtsquad_create {};
+        class virtsquad_createMissionSquadsInfo {};
+        class virtsquad_delete {};
+        class virtsquad_despawn {};
+        class virtsquad_destroyGroup {};
+        class virtsquad_getMissionSquadsInfo {};
+        class virtsquad_perFrame {};
```

Comment:
Removing the cfgfunctions header will make it little bit faster
```suggestion
        class virtsquad_perFrame {
            headerType = -1;
        };
```